source/slang/slang-emit-glsl.cpp

```diff
@@ -2068,7 +2068,9 @@ bool GLSLSourceEmitter::tryEmitInstExprImpl(IRInst* inst, const EmitOpInfo& inOu
 
             emitOperand(inst->getOperand(0), leftSide(outerPrec, prec));
             m_writer->emit("._data[");
+            m_writer->emit("uint(");
```

Comment:
Let's add a comment here on glsl not supporting anything but 32bit int, so we don't forget why we have the logic here.

Reply:
done.